be/src/main/java/codesquad/todo/history/controller/HistoryRestController.java

```diff
@@ -1,15 +1,35 @@
 package codesquad.todo.history.controller;
 
+import java.util.Collections;
+import java.util.List;
+
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
 import org.springframework.web.bind.annotation.RestController;
 
+import codesquad.todo.history.controller.dto.HistoryDeleteRequest;
+import codesquad.todo.history.controller.dto.HistoryFindAllResponse;
 import codesquad.todo.history.service.HistoryService;
 
 @RestController
+@RequestMapping("/histories")
 public class HistoryRestController {
-
 	private final HistoryService historyService;
 
 	public HistoryRestController(HistoryService historyService) {
 		this.historyService = historyService;
 	}
+
+	@GetMapping
+	public ResponseEntity<List<HistoryFindAllResponse>> getHistories() {
```

Comment:
```suggestion
	public List<HistoryFindAllResponse> getHistories() {
```
위와 같이 변경하여도 동일한 결과가 나올것입니다.

Reply:
반영했습니다. 감사합니다.